lib/package-data.js

```diff
@@ -95,10 +99,22 @@ module.exports = class PackageData {
 		return Promise.resolve()
 			.then(() => {
 				const getPublicOrganisationRepositories = githubPublicOrganisationRepositories(this.options.githubToken);
-				return getPublicOrganisationRepositories('financial-times');
+				return Promise.all(this.options.githubOrganisations.map(githubOrganisation => {
+					return getPublicOrganisationRepositories(githubOrganisation);
+				}));
 			})
 			.then(packages => {
-				this.data = packages;
+				this.data = [].concat(...packages).sort((a, b) => {
```

Comment:
Could we use `Array.from` here instead of `[].concat(...`. It would make the code simpler to read.

Reply:
Could we also name the sort function to something like `sortAphabetically`?